tests/suites/test_suite_hkdf.function

```diff
@@ -38,8 +38,8 @@ void test_hkdf( int md_alg, char *hex_ikm_string, char *hex_salt_string,
     TEST_ASSERT( ret == 0 );
 
     // Run hexify on it so that it looks nicer if the assertion fails
-    hexify( okm_string, okm, okm_len );
-    TEST_ASSERT( !strcmp( (char *)okm_string, hex_okm_string ) );
+    hexify( okm_hex, okm, okm_len );
+    TEST_ASSERT( !strcmp( (char *)okm_hex, hex_okm_string ) );
```

Comment:
As far as I see, `okm_hex` is not null-terminated if the length of okm is 128 Bytes, so `strcmp` might overflow. Increasing the size of `okm_hex` to `256` would fix that.

Reply:
`hex_okm_string` is null-terminated, so the `strcmp` should stop on that, but I see your point, I'll change

Reply:
@RonEld When `strcmp` finds the null-terminator in `hex_okm_string`, it would still need to check that the next byte in `okm_hex` is also NULL. So there's an overflow of 1 Byte here, potentially.

Reply:
yes, it's an overflow read, but not an overflow in writing, but this still needs to be addressed. Thanks!